components/production/editor/ProductionEditor.vue

```diff
@@ -2,50 +2,134 @@
   <div class="space-y-2">
     <card title="Basic Details">
       <div class="space-y-4">
-        <form-label>
+        <form-label :errors="errors" name="name">
           Name <required-star />
           <t-input
-            v-model="editingProduction.name"
+            :value="name"
             placeholder="e.g. My Show"
+            @input="$emit('update:name', $event)"
           />
         </form-label>
-        <form-label>
+        <p v-if="computedSlug">
+          <template v-if="!changingSlug">
+            Your production will be at
+            {{
+              $router.resolve({ path: `/productions/${computedSlug}` }).route
```

Comment:
if we are making it easier for production teams can this be the full link and include uobtheatre.com so they can copy and paste it...

Reply:
At this stage, the url might not be accurate (we are just guessing that the API will accept the slug they request / we compute). We probably only want to invite them to copy once computedSlug is the same as production.slug. 

Do you want to try and add such a button here? You could also add a button in pages/administration/productions/_productionSlug/index.vue, attached to the go to public page button
![image](https://user-images.githubusercontent.com/17804618/147926446-57c0a66f-e1f7-49f1-ac70-8be0434fba56.png)